internal/providers/terraform/aws/testdata/dx_connection_test/dx_connection_test.golden

```diff
@@ -16,7 +16,4 @@
  OVERALL TOTAL                                                                       $959.20 
 ──────────────────────────────────
 3 cloud resources were detected:
-∙ 3 were estimated, 3 include usage-based costs, see https://infracost.io/usage-file
-Logs:
-
-level=warning msg="Skipping resource aws_dx_connection.my_dx_connection_usage usage cost: Outbound data transfer. Could not find mapping for region does_not_exist"
```

Comment:
This warning now goes away because we're loading the values into the `RegionsUsage` struct. I don't think this is a big issue, but something we may want to add validation for at some point.